### ISSUE TITLE
Load Data in the Digital Twin

### ISSUE THREAD

**Issue by mjmafra1986** [qgisred/QGISRed#119]
How to load data obtained from the system into the hydraulic model?

**Comment by fmartine** [comment]
What do you mean by  'data obtained from the system'?   Are you trying to distribute the demands to nodes from punctual demands or from service connections?

**Comment by mjmafra1986** [comment]
No, I am referring to data from the water supply system, such as flow and pressure, obtained from the network using flow and pressure loggers, for comparative purposes with the values ​​obtained through simulation.

**Comment by fmartine** [comment]
Dear mjmafra1986,

QGISRed has an option called 'Load Field Data' on the Digital Twin menu that opens a dialogue to import the field data linked to the meters previously declared with the 'Add meter' option from the same menu. The data must be dated and saved in one or several files in sequential or table format.

At this time, the uploaded field data cannot be viewed or compared with the model results with QGISRed. They can only be exported by clicking on the 'Export field data files' option in the dialogue shown with the 'Export to INP' option of the Running menu, in order to make a comparison between the field data and the data calculated in the EPANET environment.

In the future we will provide some usage examples and enhance this option to offer more analysis capabilities and evaluate the differences between the model and reality, but that will be in the medium term.

Thank you for using QGISRed and showing interest in this capability that will increase QGISRed's features in the future.

PD. Use any translator to convert this answer into Portuguese.  Obrigado.
